### PR TITLE
Peter/ntier ra update

### PR DESCRIPTION
**Pull request by Petertaylor9999**
@telmosampaio

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/virtual-machines-linux/n-tier.md](https://github.com/mspnp/architecture-center/blob/peter/ntier-ra-update/docs/reference-architectures/virtual-machines-linux/n-tier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-linux/n-tier?branch=pr-en-us-292) |
[docs/reference-architectures/virtual-machines-windows/n-tier.md](https://github.com/mspnp/architecture-center/blob/peter/ntier-ra-update/docs/reference-architectures/virtual-machines-windows/n-tier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/n-tier?branch=pr-en-us-292) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201711152056022558-292%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5ekGLK8RVjgs9u6XyMtq2hzDHCVS02eIVIN3Tlngn9g%3D&st=2017-11-15T20%3A52%3A41Z&se=2017-12-16T20%3A57%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference-architectures/virtual-machines-linux/n-tier.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This reference architecture shows a set of proven practices for running Linux vi
 
 There are many ways to implement an N-tier architecture. The diagram shows a typical 3-tier web application. This architecture builds on [Run load-balanced VMs for scalability and availability][multi-vm]. The web and business tiers use load-balanced VMs.
 
-* **Availability sets.** Create an [availability set][azure-availability-sets] for each tier, and provision at least two VMs in each tier.  This makes the VMs eligible for a higher [service level agreement (SLA)][vm-sla] for VMs.
+* **Availability sets.** Create an [availability set][azure-availability-sets] for each tier, and provision at least two VMs in each tier.  This makes the VMs eligible for a higher [service level agreement (SLA)][vm-sla] for VMs. You can deploy a single VM in an availability set, but the single VM will not qualify for a SLA guarantee unless the single VM is using Azure Premium Storage.
```
Comment (MikeWasson) [inline docs/reference-architectures/virtual-machines-linux/n-tier.md]: should it be "an SLA"? 

To be very explicit do you want to add "... for all OS and data disks"? (per the SLA doc)

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/virtual-machines-linux/n-tier.md](https://github.com/mspnp/architecture-center/blob/peter/ntier-ra-update/docs/reference-architectures/virtual-machines-linux/n-tier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-linux/n-tier?branch=pr-en-us-292) |
[docs/reference-architectures/virtual-machines-windows/n-tier.md](https://github.com/mspnp/architecture-center/blob/peter/ntier-ra-update/docs/reference-architectures/virtual-machines-windows/n-tier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/n-tier?branch=pr-en-us-292) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201711211858249909-292%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YdDAUwTjNyaphAf65IU0QzbYYy0BtgGBWOY8Igb1ZEo%3D&st=2017-11-21T18%3A54%3A16Z&se=2017-12-22T18%3A59%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Petertaylor9999** [comment]
@telmosampaio

**Pull request merged**